model-optimizer/extensions/ops/select_test.py

```diff
@@ -188,7 +197,7 @@ def test_select_infer_condition_with_value(self, else_data_shape, than_data_shap
 
         graph_ref = build_graph_with_attrs(nodes_with_attrs=self.nodes, edges_with_attrs=self.edges,
                                            update_nodes_attributes=[
-                                               ('condition_data', {'shape': np.array(else_data_shape),
+                                               ('condition_data', {'shape': np.array(select_output_shape),
```

Comment:
Why "condition_data" shape is equal to the "select_output_shape"?

Reply:
I don't see the response to my question and the code has not changed, but the issue is resolved. Please, comment why it is resolved.

Reply:
Condition data value was updated. I just forgot update shape. It was updated on my last commit.